Resources/Prototypes/Entities/Buildings/Doors/airlock_base.yml

```diff
@@ -49,16 +49,13 @@
 - type: entity
   id: AirlockGlass
   parent: Airlock
-  name: Glass Airlock
+  name: Glass airlock
```

Comment:
I mean if you're gonna rename this name to be lower case at least do it fully..

Reply:
didnt we agreed on using upper case for the first word and lower case for the rest when naming?

Reply:
Uh, no?